gears/src/types/base/coin/decimal.rs

```diff
@@ -121,3 +121,9 @@ impl FromStr for DecimalCoin {
         Ok(Self { denom, amount })
     }
 }
+
+impl std::fmt::Display for DecimalCoin {
```

Comment:
It seems like you can implement it once for the struct `Coins`

Reply:
I may, but for now there is no need for it. It will do this in separate PR